### DISCUSSION TITLE
Why is there no way to see recent stars on my repositories anymore?

### DISCUSSION THREAD

**Discussion by phil294** [community/community#66161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

come *on* guys, what are you doing...

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @phil294 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit with your question--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
